packages/components/src/types/variants.ts

```diff
@@ -0,0 +1,14 @@
+import { Static, Type } from "@sinclair/typebox"
+
+export const CollectionVariantSchema = Type.Union([
+  Type.Literal("blog"),
+  Type.Literal("collection"),
+])
+
+export type CollectionVariant = Static<typeof CollectionVariantSchema>
+
+export interface IsomerLayoutVariants {
+  collection: {
+    variant: CollectionVariant
```

Comment:
to clarify my understanding, this variant key will be stored in _meta of a collection?

Reply:
yeap exactly!